hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -390,24 +401,28 @@ private void processPaths(JobConf job, CombineFileInputFormatShim combine, List<
   /**
    * HiveFileFormatUtils.getPartitionDescFromPathRecursively is no longer available since Hive 3.
    * This method is to make it compatible with both Hive 2 and Hive 3.
+   *
    * @param pathToPartitionInfo
    * @param dir
    * @param cacheMap
    * @return
    * @throws IOException
    */
-  private static PartitionDesc getPartitionFromPath(Map<Path, PartitionDesc> pathToPartitionInfo, Path dir,
+  private static PartitionDesc getPartitionFromPath(Map<Path, PartitionDesc> pathToPartitionInfo,
+      Path dir,
       Map<Map<Path, PartitionDesc>, Map<Path, PartitionDesc>> cacheMap)
       throws IOException {
     Method method;
     try {
-      Class<?> hiveUtilsClass = Class.forName("org.apache.hadoop.hive.ql.io.HiveFileFormatUtils");
+      Class<?> hiveUtilsClass = Class.forName("org.apache.hudi.hadoop.utils.HudiHiveFileFormatUtils");
       try {
         // HiveFileFormatUtils.getPartitionDescFromPathRecursively method only available in Hive 2.x
-        method = hiveUtilsClass.getMethod("getPartitionDescFromPathRecursively", Map.class, Path.class, Map.class);
+        method = hiveUtilsClass
+            .getMethod("getPartitionDescFromPathRecursively", Map.class, Path.class, Map.class);
       } catch (NoSuchMethodException e) {
         // HiveFileFormatUtils.getFromPathRecursively method only available in Hive 3.x
-        method = hiveUtilsClass.getMethod("getFromPathRecursively", Map.class, Path.class, Map.class);
+        method = hiveUtilsClass
+            .getMethod("getFromPathRecursively", Map.class, Path.class, Map.class);
```

Comment:
`hiveUtilsClass` is `HudiHiveFileFormatUtils` now, is the reflection working?

Reply:
Need to test if this works with hive3.

Reply:
I will modify the hiveUtilsClass in this exception back to hive3 later